app/merchant/views.py

```diff
@@ -116,6 +125,46 @@ def add_to_cart():
     return redirect(url_for('.listing_view_all'))
 
 
+@merchant.route('/add_quantity_to_cart')
+@login_required
+@merchant_required
+def add_quantity_to_cart():
```

Comment:
I very much like the idea of having a single add-to-cart function that takes a listingID and quantity. Incidentally, I created a pretty similar function for the front-end, `updateCartItem`. The next step is to fold `change_in_cart` (the function being used elsewhere to add to cart) into `add_quantity_to_cart`. 

We shouldn't use a GET for adding to the cart; a PUT (possibly a POST) would be most appropriate, since we're changing the resource, not just fetching information on it w/o side-effects. 

We should also make sure that a listing is available when we're adding it to the cart. Currently we can use the endpoint to add unavailable items to a cart.